To-do list/Controllers/TaskListViewController.swift

```diff
@@ -108,20 +109,18 @@ extension TaskListViewController: UITableViewDataSource {
         else {
             fatalError()
         }
-        tasks = tasks.sorted(by: { $0.status > $1.status })
-        users[userIndex].task = tasks
-        UserDefaultsManager.shared.setValueForUser(value: users)
+        tasks = tasks.sorted(by: { $0.status!.rawValue > $1.status!.rawValue })
         cell.titleLabel.text = tasks[indexPath.row].title
         cell.descriptionLabel.text = tasks[indexPath.row].description
-        cell.deadlineLabel.text = tasks[indexPath.row].deadline
-        cell.statusButtonOutlet.setTitle(tasks[indexPath.row].status, for: .normal)
+        cell.deadlineLabel.text = DateFormatter().string(from: tasks[indexPath.row].deadline ?? Date())
```

Comment:
Add extension of Date class to covert Date to String.

Reply:
done, extension Date added

Reply:
Date selector will be done in next PR form Anastasia